app/models/order.rb

```diff
@@ -148,7 +148,9 @@ def add_tickets_without_capacity_checks(valid_voucher, number, seats=[])
     raise Order::NotPersistedError unless persisted?
     new_vouchers = VoucherInstantiator.new(valid_voucher.vouchertype).from_vouchertype(number)
     new_vouchers.each_with_index do |v,i|
-      v.seat = seats[i] unless seats.empty?
+      if valid_voucher.showdate != nil
+        v.seat = seats[i] unless seats.empty?
```

Comment:
this should also be conditioned on showdate, since assigning a seat without a showdate makes no sense

Reply:
Hi Prof. Fox, I've submitted a new commit "fix" b084f2d and was wondering if that solves the issue you commented above?